account_fiscalyear_close/wizard/account_fiscalyear_close_wizard.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018, Jarsa Sistemas, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+
+
+class AccountFiscalYearWizard(models.TransientModel):
+    _name = 'account.fiscalyear.close.wizard'
+
+    date_start = fields.Date(
+         default=lambda self: self.get_date_start(),
+         required=True,)
+    date_end = fields.Date(
+        default=lambda self: self.get_date_end(),
+        required=True,)
+    account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="Account",
+        required=True,
+    )
+    journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string="Journal",
+        domain=[('type', '=', 'general')],
+        required=True,
+    )
+
+    @api.model
+    def get_date_start(self):
+        return str(fields.date.today().year - 1)+"-01-01"
```

Comment:
When you concatenate a string use spaces to separate each part of the concatenation Eg.

`return str(fields.date.today().year - 1) + "-01-01"`

Reply:
ok